### ISSUE TITLE
Enhancement(Posts): Allow for backdating of posts

### ISSUE THREAD

**Issue by JacobCoffee** [polarsource/polar#2885]
## Description

Let users select past dates in the Posts -> Publish -> Publishing -> Schedule option

## How

Right now, if you select a prior-to-now date it just jumps back to today.

## Why

I don't like our current platform, and would prefer to abandon it and point `blog.litestar.dev` to the Polar.sh equivalent. (side bar, it'd be cool to be able to have those hosted under that subdomain 😉)

To do this, all I need to have is the ability to backdate posts.
With that done, I can basically copypasta all previous posts into the relevant Polar organization.

**Comment by birkjernstrom** [comment]
Thank you @JacobCoffee for the excellent bug report.

What OS/Browser are you using? What you described as the solution you tried for this is indeed how we've intended for it to be solved. Just tried it out again and it works for me (macOS with Chrome).

Re: Custom domains, e.g `blog.litestar.dev`. We actually have support for it from https://github.com/polarsource/polar/issues/2632 except for exposing setup instructions & settings for it. However, I'd be happy to help you get this setup once we've solved this bug to make sure you could move everything over first.

**Comment by JacobCoffee** [comment]
Firefox/macOS. Let me try again to make sure I'm not dumb!

**Comment by JacobCoffee** [comment]
ahh! It works for new posts as expected.. sorry about that.

I was trying to change the date on an already-published post.

**Comment by birkjernstrom** [comment]
No worries at all @JacobCoffee - glad it worked :)
